cardano-cli/src/Cardano/CLI/EraBased/Run/Query.hs

```diff
@@ -1614,3 +1608,16 @@ requireEon minEra era =
   hoistMaybe
     (QueryCmdLocalStateQueryError $ mkEraMismatchError NodeEraMismatchError { nodeEra = era, era = minEra })
     (forEraMaybeEon era)
+
+-- | The output format to use, for commands with a recently introduced --output-[json,text] flag
+-- and that used to have the following default: --out-file implies JSON,
+-- output to stdout implied text.
+newOutputFormat :: Maybe QueryOutputFormat -> Maybe a -> QueryOutputFormat
+newOutputFormat format mOutFile =
+  case (format, mOutFile) of
+    (Just f, _) -> f -- Take flag from CLI if specified
+    (Nothing, Nothing) -> QueryOutputFormatText -- No CLI flag, writing to stdout: write text
+    (Nothing, Just _) -> QueryOutputFormatJson -- No CLI flag, writing to a file: write JSON
+
+textToLazyBytestring :: Text -> LBS.ByteString
```

Comment:
What about: https://hackage.haskell.org/package/text-2.1.1/docs/Data-Text-Lazy-Encoding.html#v:encodeUtf8

Reply:
This doesn't work, because my function converts _strict_ Text to a lazy bytestring.

So I changed the function's name to `strictTextToLazyBytestring` to make that more apparent. It was misleading :+1: